src/ir.h

```diff
@@ -135,6 +141,9 @@ struct Expr {
   static Expr* CreateBrTable(VarVector* targets, Var default_target);
   static Expr* CreateCall(Var);
   static Expr* CreateCallIndirect(Var);
+  static Expr* CreateCatch(Var);
+  static Expr* CreateCatchAll();
+  static Expr* CreateCatchBlock(Expr* catch_, Expr* first);
```

Comment:
Maybe nicer to just have Catch and CatchAll each be a block? Is there a benefit to having a generic CatchBlock expr?